experiments/functional/zfs-LocalPV/zfspv-clone/test.yml

```diff
@@ -0,0 +1,229 @@
+- hosts: localhost
+  connection: local
+
+  vars_files:
+    - test_vars.yml
+    - /mnt/parameters.yml
+    
+  tasks:
+    - block:
+
+        ## Generating the testname for deployment
+        - include_tasks: /utils/fcm/create_testname.yml
+
+        ## RECORD START-OF-TEST IN LITMUS RESULT CR
+        - include_tasks: "/utils/fcm/update_litmus_result_resource.yml"
+          vars:
+            status: 'SOT'
+        
+        - block:
+            - name: Check if the snapshot is present to create clone
+              shell: >
+                kubectl get volumesnapshot.snapshot -n {{ app_ns }}
+              args:
+                executable: /bin/bash
+              register: snapshot_status
+              failed_when: snapshot_name not in snapshot_status.stdout
+
+            - block:
+              - name: Update the {{ app_name }} deployment yaml with test specific values
+                template:
+                  src: busybox.j2
+                  dest: busybox.yml
+
+              - name: Deploy the {{ app_name }} application using cloned PVC
```

Comment:
How cloned pvc is created?

Reply:
Changed the sequence of tasks.
Now first we are creating clone-pvc which will remain in pending state due to WaitForFirstConsumer.
Then will deploy the application and will check the bound status of pvc and running status of application.
@gprasath